src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDb.java

```diff
@@ -103,6 +95,4 @@ public String toString() {
         }
     }
 
-    default void invalidateCaches() {}
-
```

Comment:
Unused declaration

Reply:
is there a reason we aren't ever calling any of these `invalidateCaches` methods?

Reply:
It's a feature put in for testing purposes that is no longer required.